### ISSUE TITLE
App Store

### ISSUE THREAD

**Issue by L0n0n0k** [bk20dev/forest#3]
Request to make the app available in f-droid or izzyOndroid.

**Comment by Vishal040393** [comment]
Yes, I also request you to do the same. For a non-expert, it becomes difficult to access the reliability of an app.

**Comment by candroid-man** [comment]
I agree, mostly because I had no idea this app existed until I watched [HowToMen's video](https://www.youtube.com/watch?v=urDNwLGMouM) on it.
